hs-init.hs

```diff
@@ -627,15 +630,15 @@ createStackTemplate
     copyright:           $year $nm
     category:            $category
     build-type:          Simple
-    extra-source-files:  README.md
-    cabal-version:       >=1.10
+    extra-doc-files:     README.md
+    cabal-version:       >=1.24
     $testedWith
     $endLine
     |]
 
   testedWith :: Text
-  testedWith = "tested-with:         " <>
-          T.intercalate ", " (map ("GHC == " <>) testedVersions)
+  testedWith = "tested-with:         GHC == 8.2.2" <>
```

Comment:
I think we should have `defaultGHC` variable with value `8.2.2` and use it here. When we need to bump up to `GHC-8.4.1` we just change single variable instead of remembering what we need to change.